pkgs/applications/misc/gImageReader/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv , fetchFromGitHub, cmake , pkgconfig, libuuid
+, sane-backends, podofo, libjpeg, djvulibre, libxmlxx3, libzip, tesseract
+, enchant, intltool, poppler, json-glib
+# needs to be python3
+, python
+
+# Gtk deps
+, gtk3, gobjectIntrospection, wrapGAppsHook
+, gnome3, gtkspell3, gtkspellmm, cairomm
+}:
+
+let
+  variant = "gtk";
+  pythonEnv = python.withPackages( ps: with ps;[ pygobject3 ] );
+in
+stdenv.mkDerivation rec {
+  name = "gImageReader-${version}";
+  version = "3.2.99";
+
+  src = fetchFromGitHub {
+    owner= "manisandro";
+    repo = "gImageReader";
+    rev = "v${version}";
+    sha256 = "19dbxq83j77lbvi10a8x0xxgw5hbsqyc852c196zzvmwk3km6pnc";
+  };
+  buildInputs = [
+    pythonEnv
+    cmake
+    enchant
+    intltool
+    libxmlxx3
+    libzip
+    libuuid
+    pkgconfig
+    sane-backends
+    podofo
+    libjpeg
+    djvulibre
+    tesseract
+    poppler.dev
```

Comment:
thought it would diminish closure size ? or is nixpkgs smart enough to not require this ?

Reply:
IIRC this will do the same thing as adding just poppler, all outputs will be added to build closure. As for the runtime closure, Nix just greps for store locations in the produced outputs, so unless the build instructions just copy environment variables (e.g [pitivi](https://github.com/NixOS/nixpkgs/blob/747450ef4927078376984947be2a2679699faa5b/pkgs/applications/video/pitivi/prevent-closure-contamination.patch)) it should be alright.